packages/components/src/vue/components/morph-parts/principal-parts.vue

```diff
@@ -7,18 +7,14 @@
             class="alpheios-principal-parts__groupitem"
             :lang="languageCode"
             v-if="! lemma.principalParts.includes(lemma.word)"
-        >
-            {{ lemma.word }}
-        </h4>
+        >{{lemma.word}}</h4>
```

Comment:
Here it was removing spaces around the groups of items that was not needed (on my opinion) because there already was enough space there and multiple spaces are shown as a single space by HTML anyway:
![image](https://user-images.githubusercontent.com/18631055/79895612-39154c00-8418-11ea-8876-8956f3c59f0a.png)
I thought it was a mistake in formatting. But I can return it back for safety.

Reply:
I could be wrong, I'm not sure. It just felt like something I had "fixed" before.

Reply:
> I could be wrong, I'm not sure. It just felt like something I had "fixed" before.

I'm wondering if this could be a side effect of auto-formatting in an editor or maybe result of some ESLint rules applied. But if it will start showing incorrectly as a result of this, we could change it, that is not hard.

Reply:
yes, agree.